app/components/SearchHistoryItem/__tests__/SearchHistoryItem.test.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { render } from 'react-testing-library';
+import { ThemeProvider } from '@datapunt/asc-ui';
+import 'jest-styled-components';
+import { BrowserRouter } from 'react-router-dom';
+import SearchHistoryItem from 'components/SearchHistoryItem';
+
+const URL_PROP = '/foo-url/';
+const TEXT_PROP = 'foo-text';
+
+const getRenderedComponent = (url, text) =>
```

Comment:
Might be handy to make this function a reusable one.

Reply:
It's already in a function that is being re-used. Not sure if we can make it even more re-usable?

Reply:
True, but I was thinking about something like this: https://github.com/Amsterdam/signals-frontend/blob/develop/src/test/utils.js#L48.

Reply:
Makes sense! It will only be useful if we are consistent and update all tests to use one re-usable render function. 
That's beyond the scope of this story, so I'll add a technical debt item on the backlog for this.